catalog/MaterialCatalog/MDCCatalogTiles.m

```diff
@@ -942,8 +942,17 @@ void MDCCatalogDrawNavigationBarTile(CGRect frame, id<MDCColorScheme> colorSchem
       NSMutableParagraphStyle* labelStyle = [NSMutableParagraphStyle new];
       labelStyle.alignment = NSTextAlignmentCenter;
 
+      UIFont *font;
+      if ([UIFont respondsToSelector:@selector(systemFontOfSize:weight:)]) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+        font = [UIFont systemFontOfSize:11 weight:UIFontWeightMedium];
+#pragma clang diagnostic pop
+      } else {
+        font = [UIFont fontWithName:@"HelveticaNeue" size:11];
```

Comment:
Do we usually use "HelveticaNeue" rather than systemFontOfSize (without the weight)

Reply:
Not sure, I stole this code from other places where we perform an if-else on the selector.

Reply:
Ahhh... but I missed the part where we just call `[UIFont systemFontOfSize:size]` for regular font weights. :-\

Reply:
Done